src/components/HOC/PageWrapper/PageWrapper.js

```diff
@@ -39,20 +39,23 @@ const PageWrapper = (props) => {
     inlineHeader,
     fullWidth,
     title,
-    titleProps
+    titleProps,
+    backBtnLeft
```

Comment:
add documentation for backBtnLeft and add one to PropTypes, please

Reply:
Look at the last commit changes